.env.sample

```diff
@@ -26,4 +26,5 @@ SUPPORT_WEBCHAT_CONTACT=1
 SUPPORT_PHONE_CONTACT=1
 SHOW_CONTACT_GUIDANCE=1
 CONTACT_EMAIL_SERVICE_URL=https://signin.staging.account.gov.uk/contact-us-from-triage-page
-AUDIT_QUEUE_URL=
+AUDIT_QUEUE_URL=http://localhost:4566/000000000000/localstack-queue
+AWS_REGION=us-east-1
```

Comment:
Why is the being set?  The provision.sh script does all its work in eu-west-2 (which we don't really need for localstack) which is at odds with this setting.

Reply:
Forgot to update after merging your changes. Deleted now.